collection_requirements.rst

```diff
@@ -0,0 +1,145 @@
+*****************************
+Ansible Collections Checklist
+*****************************
+
+.. contents:: Topics
+
+Overview
+========
+
+**Warning:** Subject to frequent updates
+       This is a "living document", expect it to change as we progress with the Collections work over the next few months.
+
+Feedback
+========
+
+As with any project it's very important that we get feedback from users, contributors and maintainers. We recognize that the move to Collections is a big change in how Ansible is developed and delivered.
+
+Please raise feedback by
+
+* Discussing in ``#ansible-community`` in Freenode IRC
+* Adding to the `Community Working Group IRC meeting <https://github.com/ansible/community/issues/539>`_
+* Creating `GitHub Issues <https://github.com/ansible-collections/overview/issues>`_ against this repo
+
+Be sure you're subscribed to:
```

Comment:
I'd put this part in a dedicated section, about how to be informed of changes and move before the feedback.